Matrix.SynapseInterop.Worker.FederationSender/Program.cs

```diff
@@ -38,7 +27,8 @@ private static void Main(string[] args)
                                            metricConfig.GetValue<string>("bindHost"));
 
             new FederationSender(_config).Start().Wait();
-            Thread.Sleep(-1);
+
+            Console.ReadKey(true);
```

Comment:
This breaks in docker for me :(

Reply:
how so?

Reply:
I can't remember the precise error, but it throws an exception because it can't talk to stdin.

Reply:
Works fine for me:

```
docker build -t synapseinterop:devworker -f Matrix.SynapseInterop.DevWorker/Dockerfile .
docker run -it --rm -e Synapse_replicationHost=x.x.x.x synapseinterop:devworker
```